docs/community/ecosystem.md

```diff
@@ -0,0 +1,72 @@
+* [The Straight Dope](https://gluon.mxnet.io/) - a series of notebooks designed to teach deep learning using the Gluon Python API for MXNet.
+
+
+## MXNet APIs
+
+* [Clojure API](https://github.com/apache/incubator-mxnet/tree/master/contrib/clojure-package) - use MXNet with Clojure.
+* [C++ API](../api/c++/index.html) - not be confused with the C++ backend, this API allows C++ programmers to train networks in C++.
+* [Gluon Python Interface](../gluon/index.html) - train complex models imperatively and then deploy with a symbolic graph.
+* [Julia API](../api/julia/index.html) *(Community Supported)* - train models with multiple GPUs using Julia.
+* [Keras-MXNet](https://github.com/awslabs/keras-apache-mxnet) - design with Keras2 and train with MXNet as the backend for 2x or more speed improvement.
+* [MinPy](https://github.com/dmlc/minpy) - Pure numpy practice with third party operator integration and MXNet as backend for GPU computing
+* [Module Python API](../api/python/index.html) - backed by the Symbol API, you can define your network in a declarative fashion.
+* [ONNX-MXnet API](../api/python/contrib/onnx.html) - train and use Open Neural Network eXchange (ONNX) model files.
+* [Perl API](../api/perl/index.html) *(Community Supported)* - train models with multiple GPUs using Perl.
+* [R API](https://mxnet.incubator.apache.org/api/r/index.html) *(Community Supported)* - train models with multiple GPUs using R.
+* [Scala Infer API](../api/scala/infer.html) - model loading and inference functionality.
+* [TensorFuse](https://github.com/dementrock/tensorfuse) - Common interface for Theano, CGT, TensorFlow, and MXNet (experimental) by [dementrock](https://github.com/dementrock)
+
+
+## Toolkits to Extend MXNet
+
+* [NVIDIA DALI](https://github.com/NVIDIA/DALI/tree/master/examples/mxnet) - a collection of highly optimized building blocks and an execution engine to accelerate input data pre-processing for deep learning applications.
+* [Gluon CV](https://gluon-cv.mxnet.io/) - state-of-the-art deep learning algorithms in computer vision.
+* [Gluon NLP](https://gluon-nlp.mxnet.io/) - state-of-the-art deep learning models in natural language processing.
+* [Sockeye](https://github.com/awslabs/sockeye) - a sequence-to-sequence framework for Neural Machine Translation
+
+
+## Debugging and Visualization
+
+* [MXBoard](https://github.com/awslabs/mxboard) - lets you to visually inspect and interpret your MXNet runs and graphs using the TensorBoard software.
+
+
+## Model Serving
+
+* [MXNet Model Server (MMS)](https://github.com/apache/incubator-mxnet/tree/master/example/model-server/mms.md) - simple yet scalable solution for model inference.
+
+
+## Model Zoos
+
+* [Gluon Model Zoo](https://mxnet.incubator.apache.org/api/python/gluon/model_zoo.html) - models trained in Gluon and available through Gluon's model zoo API.
+* [ONNX Model Zoo](https://github.com/onnx/model-zoo) - ONNX models from a variety of ONNX-supported frameworks.
```

Comment:
This link redirects to a personal repo and doesn't seem to be the official model zoo. The one in onnx org is https://github.com/onnx/models

Reply:
Good catch @szha - @abhinavs95 recently moved ONNX model zoo into onnx/models.